### DISCUSSION TITLE
Issue labels

### DISCUSSION THREAD

**Discussion by scottdorman** [dotnet/csharplang#54] [General]
For consistency, we should keep the issue labels in both repos the same. This would include both names and colors. Right now, they're almost the same:

### C# (https://github.com/dotnet/csharplang/labels)
- `Meeting notes issue`
- `Proposal champion`
- `Proposal issue`
- `Spec defect`

### VB (https://github.com/dotnet/vblang/labels)
- `Design-Meeting-Notes`
- `Proposal`
- `Scenario`
- `Spec-Defect`

![2017-02-10 08_52_10-labels dotnet_csharplang - microsoft edge](https://cloud.githubusercontent.com/assets/2532850/22829055/837f4850-ef6e-11e6-8cdb-b6e5cfbff35a.png)

(See https://github.com/dotnet/vblang/issues/17 for the same issue in the VB repo.)

**Comment by jnm2** [comment]
This is good for cross-repo searches.

**Comment by scottdorman** [comment]
@jnm2 It's also good for those who contribute to both repos.

**Comment by scottdorman** [comment]
If we want additional tags, @AdamSpeight2008's [proposed list](https://github.com/dotnet/vblang/issues/8) is a good start.

**Comment by alrz** [comment]
Please keep them hyphenated so it does not require quotations in search (with a consistent capitalization).

**Comment by AdamSpeight2008** [comment]
@alrz I totally agree with that.
